.gitattributes

```diff
@@ -0,0 +1 @@
+*.pdf -text
```

Comment:
This shouldn't be necessary. A pdf document isn't a binary file. It's a text file with encoded strings in it.

Reply:
Yes, that seems correct.

AppVeyor must be setting this globally because it doesn't affect the builds on AppVeyor. Sure enough: https://github.com/appveyor/docker/commit/68e82c80e0506927e68fb3c22f09ea0d36bd8974

Reply:
I really dislike that git clients still try to change the newline character to CRLF. There's really no reason to be using that line ending at this point. It's just unnecessary lashing of the file contents, and would break the linter too I would think.

Reply:
> I really dislike that git clients still try to change the newline character to CRLF. There's really no reason to be using that line ending at this point.

You can add`* -text` to `.gitattributes`. It means "Git, stop touching my EOLs".

Reply:
I'd prefer the setting that tells git not to mess with the files. Though I could also see the risk of someone committing a file with CRLF in it. So enabling both settings might be best.

Reply:
They're mutually exclusive. Documentation: https://git-scm.com/docs/gitattributes#_checking_out_and_checking_in

Reply:
Okay, changed this to `*.pdf eol=lf`.

Reply:
It's not binary data. It's just an encoded stream.

Reply:
It's encoded in FlateDecode. I mean, it's not plain text to be sure. But it's still character data.

Reply:
I mean, it's sequence of bytes that have CRLF sequence in them and it would be an error to convert CRLF->LF in this sequence. `-text` is the only proper solution for this.